cluster/addons/fluentd-gcp/fluentd-gcp-image/google-fluentd.conf

```diff
@@ -16,14 +16,14 @@
   type tail
   format none
   time_key time
-  path /var/lib/docker/containers/*/*-json.log
-  pos_file /var/lib/docker/containers/gcp-containers.log.pos
+  path /varlog/containers/*/*.log
```

Comment:
I still hate this directory name here, but no proposal on a better one ... we can change later.